docs/zh-cn/cli_commands.md

```diff
@@ -0,0 +1,442 @@
+# QMK CLI 命令
+
+# 用户命令
+
+## `qmk compile`
+
+该命令用于在指定目录下编译固件，可用于构建<https://config.qmk.fm>导出的JSON数据，代码库中的键映射，或是当前目录下的键盘。
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**用于配置器导出的数据时**：
+
+```
+qmk compile [-c] <configuratorExport.json>
+```
+
+**用于键映射时**：
+
+```
+qmk compile [-c] [-e <var>=<value>] [-j <num_jobs>] -kb <keyboard_name> -km <keymap_name>
+```
+
+**在键盘目录下时**：  
+
+须在存在默认键映射的键盘目录下执行，或是在键盘的键映射子目录下，否则须指定参数 `--keymap <keymap_name>`
+```
+qmk compile
+```
+
+**构建所有支持该键映射的键盘时**：
+
+```
+qmk compile -kb all -km <keymap_name>
+```
+
+**示例**：
+```
+$ qmk config compile.keymap=default
+$ cd ~/qmk_firmware/keyboards/planck/rev6
+$ qmk compile
+Ψ Compiling keymap with make planck/rev6:default
+...
+```
+指定键映射参数时
+
+```
+$ cd ~/qmk_firmware/keyboards/clueboard/66/rev4 
+$ qmk compile -km 66_iso
+Ψ Compiling keymap with make clueboard/66/rev4:66_iso
+...
+```
+位于键盘目录下时
+
+```
+$ cd ~/qmk_firmware/keyboards/gh60/satan/keymaps/colemak
+$ qmk compile
+Ψ Compiling keymap with make make gh60/satan:colemak
+...
+```
+
+**在配列目录下时**：  
+
+必须是在 `qmk_firmware/layouts/` 下的键映射目录下。
+```
+qmk compile -kb <keyboard_name>
+```
+
+**示例**：
+```
+$ cd ~/qmk_firmware/layouts/community/60_ansi/mechmerlin-ansi
+$ qmk compile -kb dz60
+Ψ Compiling keymap with make dz60:mechmerlin-ansi
+...
+```
+
+**并行编译**：
+
+在编译时添加 `-j`/`--parallel` 开关可能有助于加快编译速度。
+```
+qmk compile -j <num_jobs> -kb <keyboard_name>
+```
+`num_jobs` 用于指定并行的job上限，将其设置为0可以实现无限制的并行编译。
+```
+qmk compile -j 0 -kb <keyboard_name>
+```
+
+## `qmk flash` :id=qmk-flash
+
+该命令与 `qmk compile` 类似，但额外地可以指定bootloader。bootloader参数是可选的，默认会指定为 `:flash`。可通过 `-bl <bootloader>` 来指定bootloader。请查阅[刷写固件](zh-cn/flashing.md)指引以深入了解可用的bootloader信息。
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**用于配置器导出的数据时**：
+
+```
+qmk flash [-bl <bootloader>] [-c] [-e <var>=<value>] [-j <num_jobs>] <configuratorExport.json>
+```
+
+**用于键映射时**：
+
+```
+qmk flash -kb <keyboard_name> -km <keymap_name> [-bl <bootloader>] [-c] [-e <var>=<value>] [-j <num_jobs>]
+```
+
+**列出所有bootloader**
+
+```
+qmk flash -b
+```
+
+## `qmk config`
+
+该命令用于配置QMK功能，完整的 `qmk config` 文档参见[CLI配置](zh-cn/cli_configuration.md)。
+
+**使用方法**：
+
+```
+qmk config [-ro] [config_token1] [config_token2] [...] [config_tokenN]
+```
+
```

Comment:
Missing section https://github.com/qmk/qmk_firmware/blob/20beac5e571e3cb7694a53386c5b715d1bf6eee0/docs/cli_commands.md?plain=1#L121-L133

Reply:
added in latest commits